### PR TITLE
Make template descriptions shorter

### PR DESCRIPTION
**Pull request by akrejcir**
The original descriptions were unnecessarily long.

Signed-off-by: Andrej Krejcir <akrejcir@redhat.com>

### CODE REVIEW

File: templates/fedora.tpl.yaml [hunk 1]
```diff
@@ -6,10 +6,8 @@ metadata:
   annotations:
     openshift.io/display-name: "Fedora 23+ VM"
     description: >-
-      This template can be used to create a VM suitable for
-      Fedora 23 and newer.
-      The template assumes that a PVC is available which is providing the
-      necessary Fedora disk image.
+      Template for Fedora 23 VM or newer.
```
Comment (ksimon1) [inline templates/fedora.tpl.yaml:9]: Please change this to Fedora 31 VM or newer.
Please change `openshift.io/display-name` annotation as well

Comment (akrejcir) [reply templates/fedora.tpl.yaml]: Done

File: templates/centos6.tpl.yaml [hunk 2]
```diff
@@ -6,10 +6,8 @@ metadata:
   annotations:
     openshift.io/display-name: "CentOS 6.0+ VM"
     description: >-
-      This template can be used to create a VM suitable for
-      CentOS 6 and newer.
-      The template assumes that a PVC is available which is providing the
-      necessary CentOS disk image.
+      Template for CentOS 6 VM or newer.
+      A PVC with the CentOS disk image should be available.
```
Comment (MarSik) [inline templates/centos6.tpl.yaml:10]: I believe "must be available" is the proper form here (the same in all the other templates).

Comment (akrejcir) [reply templates/centos6.tpl.yaml]: Changed

**Comment by akrejcir** [comment]
ping? Is this PR ok now?

**Pull request merged**